test/e2e/main_test.go

```diff
@@ -307,8 +307,9 @@ func TestCacheService(t *testing.T) {
 	t.Parallel()
 	spec := DefaultSpec.DeepCopy()
 	name := strcase.ToKebab(t.Name())
-
-	spec.ObjectMeta.Name = name
+	
+	spec.Name = name
+	spec.Spec.Image = pointer.StringPtr(tconst.NativeImageName)
```

Comment:
Do we want the native image to always be used by `TestCacheService`?

It would be good if we could parametise the tests so that all of them are executed with both the normal and native image.

Reply:
Yes, it could be. But every test iteration will add additional time to CI execution. If it's ok, I will add this option.

Reply:
I think the extra test coverage would be worth it, @rigazilla any objections?

Reply:
@rigazilla I'm waiting you feedback, do we need to run test TestCacheService with standard and native images or just with native. Could you please review this PR.

Reply:
@dmvolod I agree with @ryanemerson that ideally we should run the testsuite agaist all the community released images.
Testsuite changes probably need an update, due to #614 ?

Reply:
Yes will modify test and rebase on top of latest fix